### PR TITLE
yurt ingress use helm reconcile

### PR DESCRIPTION
**Pull request by huiwq1990**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: cmd/yurt-app-manager/options/options.go [hunk 1]
```diff
@@ -64,6 +65,7 @@ func (o *YurtAppOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.EnablePprof, "enable-pprof", o.EnablePprof, "Enable pprof for controller manager.")
 	fs.StringVar(&o.LeaderElectionNamespace, "leader-election-namespace", o.LeaderElectionNamespace, "This determines the namespace in which the leader election configmap will be created, it will use in-cluster namespace if empty.")
 	fs.StringVar(&o.Namespace, "namespace", o.Namespace, "Namespace if specified restricts the manager's cache to watch objects in the desired namespace. Defaults to all namespaces.")
+	fs.StringVar(&o.WorkloadNamespace, "workloadNamespace", o.WorkloadNamespace, "")
```
Comment (rambohe-ch) [inline cmd/yurt-app-manager/options/options.go:68]: please add description for the parameter.

File: pkg/yurtappmanager/apis/apps/v1alpha2/yurtingress_types.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+// YurtIngressCondition describes current state of a YurtIngress
+type YurtIngressCondition struct {
+}
+
+// YurtIngressStatus defines the observed state of YurtIngress
+type YurtIngressStatus struct {
+
+	// Indicates all the nodepools on which to enable ingress.
+	// +optional
+	Conditions YurtIngressCondition `json:"conditions,omitempty"`
```
Comment (rambohe-ch) [inline pkg/yurtappmanager/apis/apps/v1alpha2/yurtingress_types.go:41]: Conditions YurtIngressCondition --> Conditions []YurtIngressCondition?

**Comment by rambohe-ch** [comment]
@zzguang PTAL

**Comment by huiwq1990** [comment]
use new version https://github.com/openyurtio/yurt-app-manager/pull/127

**Pull request closed without merging**